app.js

```diff
@@ -1,27 +1,27 @@
-const showNav = document.getElementById("show-nav-btn");
-const hideNav = document.getElementById("hide-nav-btn");
-const mobileMenuLinks = document.getElementById("mobile-menu-links");
-const mobileMenu = document.getElementById("mobile-menu");
+const showNav = document.getElementById('show-nav-btn');
+const hideNav = document.getElementById('hide-nav-btn');
+const mobileMenuLinks = document.getElementById('mobile-menu-links');
+const mobileMenu = document.getElementById('mobile-menu');
 
-showNav.addEventListener("click", () => {
-  mobileMenu.classList.toggle("mobile-menu--show");
+showNav.addEventListener('click', () => {
+  mobileMenu.classList.toggle('mobile-menu--show');
 });
 
-hideNav.addEventListener("click", () => {
-  mobileMenu.classList.toggle("mobile-menu--show");
+hideNav.addEventListener('click', () => {
+  mobileMenu.classList.toggle('mobile-menu--show');
 });
 
-mobileMenuLinks.addEventListener("click", () => {
-  mobileMenu.classList.toggle("mobile-menu--show");
+mobileMenuLinks.addEventListener('click', () => {
+  mobileMenu.classList.toggle('mobile-menu--show');
 });
 
-mobileMenuLinks.forEach((link) => {
-  link.addEventListener("click", (event) => {
-    event.preventDefault();
+// mobileMenuLinks.forEach((link) => {
+//   link.addEventListener("click", (event) => {
+//     event.preventDefault();
```

Comment:
- [ ] Leaving unused pieces of code as comments makes our codebase look unclean. I think it would be better to remove them to maintain a professional repository. 😎️ You can use git history to revisit old pieces of code.